ui/packages/admin-security-certificate-ui/src/main/webapp/js/view/UploadModal.view.js

```diff
@@ -218,6 +222,7 @@ define([
                 var url = this.$('.urlField').val();
                 var encodedUrl = btoa(url);
                 this.urlModel.fetch({url: this.urlModel.saveUrl + "/" + encodedUrl}).done(function(result) {
+                    that.options.collection.parents[0].fetch({reset: true});
```

Comment:
❓ It's a little odd to reach into parents (it's usually not a good sign to have to know about the parent of a model or collection).  That said, there might not be an easier route at this point.

Reply:
I don't know of an easier way.